pandas/tests/indexing/test_loc.py

```diff
@@ -1611,16 +1561,14 @@ def test_loc_getitem_label_slice_across_dst(self):
         expected = 2
         assert result == expected
 
-    def test_loc_getitem_label_slice_period(self):
-        ix = pd.period_range(start="2017-01-01", end="2018-01-01", freq="M")
-        ser = ix.to_series()
-        result = ser.loc[: ix[-2]]
-        expected = ser.iloc[:-1]
-
-        tm.assert_series_equal(result, expected)
-
-    def test_loc_getitem_label_slice_timedelta64(self):
-        ix = timedelta_range(start="1 day", end="2 days", freq="1H")
+    @pytest.mark.parametrize(
+        "ix",
+        [
+            pd.period_range(start="2017-01-01", end="2018-01-01", freq="M"),
+            timedelta_range(start="1 day", end="2 days", freq="1H"),
+        ],
+    )
+    def test_loc_getitem_label_slice_period(self, ix):
```

Comment:
rename to not period-specific?

is there a corresponding dt64 test that should be rolled into this?

call it `index` instead of `ix`?

Reply:
Done